07.high-fidelity-commerce/01.basics/01.hfc/docs.md

```diff
@@ -10,11 +10,9 @@ taxonomy:
 
 High Fidelity Coins is a cryptocurrency that is empowered by blockchain technology.
 
-We are currently giving out the currency for anyone interested in participating in the closed beta for High Fidelity Commerce. If you want to [get your inital HFC grant](/high-fidelity-commerce/basics/hfc/get-hfc), you first need to [set up your Wallet](https://docs.highfidelity.com/high-fidelity-commerce/basics/hfc/wallet-setup).
+We are currently giving out the currency for anyone interested in participating in the closed beta for High Fidelity Commerce. These coins are to be used as currency for any commerce transactions in the Marketplace. Since we are using blockchain technology, all transactions with HFCs will be publically recorded and stored. 
 
-These coins are to be used as currency for any commerce transactions in the Marketplace. Since we are using blockchain technology, all transactions with HFCs will be publically recorded and stored. 
-
-Your Wallet will be secured using ECDSA public-private key pairs, security picture and passphrase. Learn more about your Wallet [here](https://docs.highfidelity.com/high-fidelity-commerce/basics/hfc/wallet-setup). 
+Your Wallet is accessed by logging into your High Fidelity account. In addition, you will select a Security Picture, which serves as an extra layer of security for your Wallet. When you see your Security Picture, you know that your actions and data are securely making use of your account.
```

Comment:
Users are now assigned a security pic; they aren't asked to select one. They _can_ change their randomly-assigned security pic in Settings, but for now, this isn't correct.

Reply:
Fixed